src/renderer/components/NewProjectModal/RunTranscriptionView.tsx

```diff
@@ -61,21 +72,55 @@ const RunTranscriptionView = ({ closeModal, nextView }: Props) => {
     ) {
       return;
     }
+    setMediaFilePath(currentProject.mediaFilePath);
+
+    if (mediaFilePath === null) {
+      return;
+    }
+    const setCurrentProject = (project: RuntimeProject) => {
+      return dispatch(projectCreated(project));
+    };
+
+    const extractProjectAudio = async () => {
+      const audioFilePath = await extractAudio(currentProject);
+      const projectWithAudioExtract = await updateProjectWithExtractedAudio(
+        currentProject,
+        audioFilePath
+      );
+      if (projectWithAudioExtract === null) {
+        return;
+      }
+      setCurrentProject(projectWithAudioExtract);
+    };
 
     setAsyncState(AsyncState.LOADING);
 
-    requestTranscription(currentProject)
-      .then((transcription) => {
-        if (transcription === null) {
+    const startProcessing = async () => {
+      await updateProjectWithMedia(currentProject, mediaFilePath);
+      await extractProjectAudio();
+      requestTranscription(currentProject)
```

Comment:
If we're awaiting everything else, may as well refactor the `requestTranscription` call to be async as well (with a try/catch for the error state)